app/models/installed_heat_source.rb

```diff
@@ -0,0 +1,18 @@
+class InstalledHeatSource
+  include Virtus.model
+
+  attribute :key, String
+  attribute :name, String
+  attribute :units, Float
+  attribute :heat_production, Float
```

Comment:
> I'm curious about the names heat_production and heat_capacity. Are these the heat technology versions of demand and capacity?

You could say so. I think the prefix `heat_` is a bit redundant which I think is what you are trying to tell me (?). I am a bit freightened about the `capacity` term since there's an `input_`- and `output_capacity` and @ChaelKruip has a strong opinion about this matter.

> Is there a limitation in the ETEngine importer such that we have to use different names?

Nope